local-cli/runIOS/runIOS.js

```diff
@@ -25,15 +27,37 @@ function runIOS(argv, config, args) {
   const scheme = args.scheme || inferredSchemeName;
   console.log(`Found Xcode ${xcodeProject.isWorkspace ? 'workspace' : 'project'} ${xcodeProject.name}`);
 
+  if (args.device) {
+    const devices = parseIOSDevicesList(
+      child_process.execFileSync('xcrun', ['instruments', '-s'], {encoding: 'utf8'})
```

Comment:
I am pretty sure I've seen similar PR in the past and there was discussion that you can pass `json` flag?